content/applications/general/voip/devices_integrations.rst

```diff
@@ -0,0 +1,185 @@
+========================
+Devices and integrations
+========================
+
+:abbr:`VoIP (Voice over Internet Protocol)` can be used on many different devices, ranging from, a
+computer, tablet, mobile phone and many more. This is helpful in that it reduces costs, and
+employees can work from anywhere in the world with a broadband internet connection.
+
+Odoo *VoIP* is SIP (Session Initiation Protocol) compatible, which means that it can be used with
+*any* :abbr:`SIP (Session Initiation Protocol)` compatible application. This document covers setting
+up Odoo *VoIP* across different devices and integrations.
+
+Odoo is fully integrated with all Odoo apps, users can click into any app and schedule a call as an
+activity in the chatter.
+
+.. example::
+   For example, in the *CRM* app a user can click into an opportunity and click on
+   :guilabel:`Activities` in the chatter. Next, choose :guilabel:`Call` and under :guilabel:`Due
+   Date` select a date. Click :guilabel:`Save` and now in the chatter an activity shows up. Should
+   the :guilabel:`Due Date` be for today's date, then the activity shows up in the :abbr:`VoIP
+   (Voice over Internet Protocol)` widget.
+
+   .. image:: devices_integrations/crm-voip-widget.png
+      :align: center
+      :alt: View of CRM leads and the option to schedule an activity for Odoo Discuss
+
+   .. seealso::
+      :doc:`voip_widget`
```

Comment:
```suggestion
:abbr:`VoIP (Voice over Internet Protocol)` can be used on many different devices, such as a
computer, tablet, mobile phone, and many more. This is helpful in that it reduces costs, and
employees can work from anywhere in the world, so long as they have a broadband internet connection.

Odoo *VoIP* is SIP (Session Initiation Protocol) compatible, which means it can be used with *any*
:abbr:`SIP (Session Initiation Protocol)` compatible application.

This document covers the process of setting up Odoo *VoIP* across different devices and
integrations.

Odoo is fully-integrated with all Odoo apps, allowing users to click into any app, and schedule a
call as an activity in the chatter.

.. example::
   For example, in the *CRM* app, a user can click into an opportunity, and click on
   :guilabel:`Activities` in the chatter.
   
   Next, they can choose :guilabel:`Call`, and under :guilabel:`Due Date`, they can select a date.
   
   Once they click :guilabel:`Save`, an activity shows up in the chatter.
   
   Should the :guilabel:`Due Date` be for today's date, the activity shows up in the :abbr:`VoIP
   (Voice over Internet Protocol)` widget.

   .. image:: devices_integrations/crm-voip-widget.png
      :align: center
      :alt: View of CRM leads and the option to schedule an activity for Odoo Discuss

   .. seealso::
      :doc:`voip_widget`
```